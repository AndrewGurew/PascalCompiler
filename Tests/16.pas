str:='A + B * c - 3'