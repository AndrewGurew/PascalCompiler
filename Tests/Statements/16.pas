var a:integer;
begin 
if(a < 3) or (a > 0) or (a = 3) then
a:=3;
end.