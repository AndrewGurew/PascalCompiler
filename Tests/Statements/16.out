Main block
 ⎬ If-Else statement
  ⎬or
   ⎬or
   ⎪⎬<
   ⎪⎪⎬a
   ⎪⎪⎬3
   ⎪⎬>
   ⎪ ⎬a
   ⎪ ⎬0
   ⎬=
    ⎬a
    ⎬3
  ⎬ Assign
   ⎬a
   ⎬3
End of Main block
